src/lib.rs

```diff
@@ -172,3 +167,13 @@ pub trait UncompressedEncoding: Sized {
     /// the point at infinity.
     fn to_uncompressed(&self) -> Self::Uncompressed;
 }
+
+/// Obtain or determine the identity point.
```

Comment:
Same issue ("point" instead of "element").

I don't know how to describe this trait, which is concerning. It's not an extension trait because it's being added as a required bound. But the constant and method it defines do not make sense in a vacuum; they need to be in the context of a group.

This makes me question the premise of the PR: what is the motivating use case for generalising over both the affine and efficient forms of a curve point, such that a shared trait is needed?

Reply:
Apologies for the delay.

The original motivation was https://github.com/RustCrypto/traits/issues/1170. We currently use `Default` to obtain the identity element, which isn't ideal and doesn't support const.

Unfortunately this was a long time ago and I don't have all the details in my head anymore, but I assume these sort of requirements will keep popping up in `elliptic-curve` because of it's nature: it introduces types that are generic over many different *things*, so it requires traits that are equally generic to support this idea.

Maybe @tarcieri has more to say on this topic.